localgovdrupal/localgov_base
# Fixes eslint issues in back-to-top.js

Closes #680

## What does this change?

Fixes all the issues reported by Drupal core's eslint rules in the back-to-top.js file.

## How to test

Enable the 'back to top' checkbox in your theme settings.

Go to a long page.

Make sure you have a "Back to top" button when you scroll.

## How can we measure success?

Success equals not having to do this again.
